Step_Functions/Quantification_Alpha/Quantification_Asymmetry.m

```diff
@@ -211,7 +212,7 @@
                     Localisation = repmat("Frontal",1,length(Electrodes));
                     AlphaType = repmat(["left", "right"],1,length(Electrodes)/2);
                 end
-                
+   % is it correct that left and right are clustered together into one electrode name?             
```

Comment:
ASY can be defined either as Difference score (then there is only one value and hemisphere is set to "Diff"), or separately for left or right hemisphere. The value in electrode is the same for both, only difference is then hemisphere.
For the LMs I need to have the factor hemisphere (left/right/diff) + Electrode(set).

Please remove comment or rephrase